tests/management/workspace/test_view.py

```diff
@@ -1064,16 +1064,18 @@ def test_move_with_invalid_uuid_parent_id(self):
         response_body = response.json()
         self.assertEqual(response_body.get("detail"), f"{invalid_uuid} is not a valid UUID.")
 
-    def test_move_under_itself(self):
-        """Test you cannot move a workspace under itself."""
-        url = reverse("v2_management:workspace-move", kwargs={"pk": self.standard_workspace.id})
-        client = APIClient()
-        workspace_data_for_move = {"parent_id": self.standard_workspace.id}
-
-        response = client.post(url, workspace_data_for_move, format="json", **self.headers)
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-        response_body = response.json()
-        self.assertEqual(response_body.get("detail"), "Cannot move workspace under itself.")
+    # TODO: uncomment after workspace move feature will be implemented ->
```

Comment:
Consider using `@skip()` decorator instead of commenting for the two commented tests?

e.g. `@skip("pending workspace move implementation")`

Reply:
implemented